### PR TITLE
Adding support for windows

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
* Updated package.json to install custom-login and okta-login dependencies
* Moved "Running Tests" section to contributing.md
* Update the custom-login README to include note about windows support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
     "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
     "resource-server": "mvn -Dokta.oauth2.issuer=https://${OKTA_DOMAIN}.com/oauth2/default -f samples-java-spring-mvc/resource-server/pom.xml",
-    "test:e2e": "export TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
+    "test:e2e": "cross-env TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
     "get-oidc-tck": "[ ! -d okta-oidc-tck/ ] && git clone https://github.com/okta/okta-oidc-tck.git || echo \"TCK already cloned\"",
-    "pretest": "webdriver-manager update --gecko false"
+    "pretest": "webdriver-manager update --gecko false",
+    "postinstall": "npm run install-custom-login && npm run install-okta-hosted-login",
```
Comment (jmelberg-okta) [inline package.json]: On the second call here, do we need to `cd` back into our working directory?

File: custom-login/README.md [hunk 2]
```diff
@@ -52,13 +52,20 @@ Now start the app server:
 ng serve
 ```
 
+> **Note:** If you're on a windows machine, you might get an error like this
```
Comment (jmelberg-okta) [inline custom-login/README.md]: like this -> similar to:

File: custom-login/README.md [hunk 3]
```diff
@@ -52,13 +52,20 @@ Now start the app server:
...
+```
+ng serve contains error Unknown browser query `basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
```
Comment (jmelberg-okta) [inline custom-login/README.md]: Using the callout `>` and a code block makes this render a little funny.

Maybe move the code block into `back-ticks` so it fits in one section?

> blah blah
> blah
> `error goes here`
> **Note**: Big scary warning

File: package.json (hunk 1 above)
Comment (vijetmahabaleshwar-okta) [reply package.json]: Running `npm install` in the root doesn't actually `cd` into the sub-directory on the console. I think we are good here. I've testing `npm install` on mac and windows. Both look fine.

File: custom-login/README.md [hunk 4]
```diff
@@ -52,13 +52,20 @@ Now start the app server:
 ng serve
 ```
 
+> **Note:** If you're on a windows machine, you might get an error similar to
+> ```
+> ng serve contains error Unknown browser query `basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
```
Comment (jmelberg-okta) [inline custom-login/README.md]: This still renders a bit funny with the overflow text. You can see that [here](https://github.com/okta/samples-js-angular/blob/0e4d40bf6455dddd8d392ccd2bcf5ba7316c2dab/custom-login/README.md#running-this-example): 

Suggestion:

----
>  **Note:** If you're on a windows machine, you might get an error similar to
> `ng serve contains error Unknown browser query basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
>
> It's a [known issue](https://github.com/angular/angular-cli/issues/5075) on windows due to one of the dependencies we use. To resolve the issue, search for browserslist and browserslist.cmd files in your node_modules. Delete both the files and start the app server again.

**Pull request merged**